bootstrap-app/src/cmr/bootstrap/services/bootstrap_service.clj

```diff
@@ -164,7 +154,6 @@
 (defn finalize-rebalance-collection
   "Finalizes collection rebalancing."
   [context concept-id]
-  (validate-collection context (:provider-id (concepts/parse-concept-id concept-id)) concept-id)
```

Comment:
I think it makes sense to remove the validation here in finalize.

We should add a test for finalize with a collection ID that doesn't exist to be sure it doesn't need the validation. Basically the same test as the "Finalizing not started collection" test, but use an ID that doesn't exist and verify it fails with the same validation error that the concept ID is not marked as rebalancing.